scalableminds/webknossos
# Add color picker to select tree color

### URL of deployed dev instance (used for testing):
- https://colorselectionfortrees.webknossos.xyz

### Steps to test:
- Open a skeleton tracing and create a tree with some nodes.
- Then use the color selection by clicking the [eye-dropper icon](https://fontawesome.com/icons/eye-dropper?style=solid).
The color of the tree should now be the newly selected color.

### Issues:
- fixes #4794

------
